### PR TITLE
return selected indices

### PR DESCRIPTION
**Pull request by oddoking**
We would like to return the selected indecies as well when doing select with top K. Please have a look

### CODE REVIEW

File: chromax/functional.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def select(
     population: Population["n"],
     k: int,
     f_index: Callable[[Population["n"]], Float[Array, "n"]],
-) -> Population["k"]:
+) -> (Population["k"], Int[Array, "k"]):
```
Comment (younik) [inline chromax/functional.py]: Can you change `(Population["k"], Int[Array, "k"])` to be `Tuple[Population["k"], Int[Array, "k"]]`, where Tuple is imported from typing?

**Review by younik — CHANGES_REQUESTED** [review]
Hello @oddoking, thanks for the PR!

The change sounds good, can you also update the typing of Simulator.select?
Thank you

File: chromax/functional.py (hunk 1 above)
Comment (oddoking) [reply chromax/functional.py]: @younik cool, updated as your suggest. What do you think now?

File: chromax/functional.py [hunk 2]
```diff
@@ -154,8 +154,8 @@ def select(
         (n, m, 2) and returns an array of n float number.
     :type f_index: Callable
 
-    :return: output population of (k, m, d)
-    :rtype: ndarray
+    :return: output population of (k, m, d), output indecies of (k,)
```
Comment (younik) [inline chromax/functional.py]: small typo `indices`

File: chromax/functional.py (hunk 1 above)
Comment (younik) [reply chromax/functional.py]: @oddoking looks good, thanks!
There is a small typo in the doc + can you also update the typing of `Simulator.select` accordingly?

Comment (younik) [reply chromax/functional.py]: Also tests, should be updated accordingly (basically, every time there is `pop = simulator.select(...)`, we should change to `pop, _ = simulator.select(...)` to unpack args

**Comment by oddoking** [comment]
@younik thanks for the help. I push the change to fix typo and tests.

**Comment by younik** [comment]
Looks good! Many thanks @oddoking for contributing!

**Pull request merged**